CMakeLists.txt

```diff
@@ -221,27 +221,29 @@ set(NMC_VALIDATION_DATA_DIR "${PROJECT_SOURCE_DIR}/validation/data" CACHE PATH
   "location of generated validation data")
 
 #----------------------------------------------------------
-# Whether to build validation data at all
-#----------------------------------------------------------
-option(NMC_BUILD_VALIDATION_DATA "generate validation data" ON)
-
-# Whether to attempt to use julia to build validation data
-find_program(JULIA_BIN julia)
-if(JULIA_BIN STREQUAL "JULIA_BIN-NOTFOUND")
-    message(STATUS "julia not found; will not automatically build validation data sets from julia scripts")
-    set(NMC_BUILD_JULIA_VALIDATION_DATA FALSE)
-else()
-    set(NMC_BUILD_JULIA_VALIDATION_DATA TRUE)
-endif()
+# Whether to build validation data
+#----------------------------------------------------------
+# turn off by default
+option(NMC_BUILD_VALIDATION_DATA "generate validation data" OFF)
```

Comment:
Still going to merge the PR, but why turn off validation data generation by default? As things stand, I think it is good to generate it by default if we can.